src/components/Menu.vue

```diff
@@ -43,10 +43,7 @@
           <li @click="toggleNight" v-if="!session.isSpectator">
             <template v-if="!grimoire.isNight">Switch to Night</template>
             <template v-if="grimoire.isNight">Switch to Day</template>
-            <em
-              ><font-awesome-icon
-                :icon="['fas', grimoire.isNight ? 'sun' : 'cloud-moon']"
```

Comment:
This was the only place where the `sun` icon is used, would you mind removing it from the list of icons in the main.js as well?

Reply:
yep, no problem, I have removed this icon form the import in the latest commit.